internal/kms/secretskms.go

```diff
@@ -69,43 +69,43 @@ func newSecretsKMS(args ProviderInitArgs) (EncryptionKMS, error) {
 		return nil, errors.New("missing encryption passphrase in secrets")
 	}
 
-	return SecretsKMS{passphrase: passphraseValue}, nil
+	return secretsKMS{passphrase: passphraseValue}, nil
 }
 
 // Destroy frees all used resources.
-func (kms SecretsKMS) Destroy() {
+func (kms secretsKMS) Destroy() {
 	// nothing to do
 }
 
 // FetchDEK returns passphrase from Kubernetes secrets.
-func (kms SecretsKMS) FetchDEK(key string) (string, error) {
+func (kms secretsKMS) FetchDEK(key string) (string, error) {
 	return kms.passphrase, nil
 }
 
 // StoreDEK does nothing, as there is no passphrase per key (volume), so
 // no need to store is anywhere.
-func (kms SecretsKMS) StoreDEK(key, value string) error {
+func (kms secretsKMS) StoreDEK(key, value string) error {
 	return nil
 }
 
 // RemoveDEK is doing nothing as no new passphrases are saved with
-// SecretsKMS.
-func (kms SecretsKMS) RemoveDEK(key string) error {
+// secretsKMS.
+func (kms secretsKMS) RemoveDEK(key string) error {
 	return nil
 }
 
-// SecretsMetadataKMS is a KMS based on the SecretsKMS, but stores the
+// SecretsMetadataKMS is a KMS based on the secretsKMS, but stores the
 // Data-Encryption-Key (DEK) in the metadata of the volume.
 type SecretsMetadataKMS struct {
```

Comment:
```suggestion
type secretsMetadataKMS struct {
```

Unexport this struct too ?

Reply:
@Rakshith-R done.. ptal,